constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintHelper.java

```diff
@@ -46,10 +46,27 @@
      * @hide
      */
     protected int[] mIds = new int[32];
+    /**
+     * @hide
+     */
+    protected float[] mAngles = new float[32];
```

Comment:
those added variables should be in CircularFlow, not in ConstraintHelper (mAngles, mRadius, etc.)

Reply:
Done ✅